pkg/shares/split_compact_shares.go

```diff
@@ -97,9 +125,9 @@ func (css *CompactShareSplitter) stackPending() {
 }
 
 // Export finalizes and returns the underlying compact shares.
-func (css *CompactShareSplitter) Export() []Share {
+func (css *CompactShareSplitter) Export() (shares []Share, shareRanges map[coretypes.TxKey]ShareRange) {
```

Comment:
nit: Does this need to have named returns? They're not really used and the return types are relatively self explanatory

Reply:
This function doesn't need to have named returns. I like named returns because they serve as additional function signature documentation but can remove and put this information in godoc. TBH godoc should probably have been updated regardless. Do you prefer?

**Option A**
```go
// Export finalizes and returns the underlying compact shares and a map of shareRanges.
func (css *CompactShareSplitter) Export() ([]Share, map[coretypes.TxKey]ShareRange) {
```

**Option B**
```go
// Export finalizes and returns the underlying compact shares and a map of shareRanges.
func (css *CompactShareSplitter) Export() (shares []Share, shareRanges map[coretypes.TxKey]ShareRange) {
```

Reply:
Went with **Option A** in 4e557a0e016abbca3ee28ec984b5437ae2e0f176